### PR TITLE
fixed scanning alert bugs

### PR DESCRIPTION
**Pull request by MikyasDesta**
# What does this PR do ?

Fixes these issues
#5933 https://github.com/NVIDIA/NeMo/issues/5933 
#5937  https://github.com/NVIDIA/NeMo/issues/5937

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Comment by MikyasDesta** [comment]
@XuesongYang Can you approve this please?

File: nemo/collections/tts/modules/common.py [hunk 1]
```diff
@@ -628,6 +628,9 @@ def __init__(
                 kernel_size=kernel_size,
                 use_partial_padding=use_partial_padding,
             )
+        else:
+            raise Exception("{} affine model not supported".format(affine_model))
```
Comment (XuesongYang) [inline nemo/collections/tts/modules/common.py:632]: We mostly use f-string, i.e. `f"Affine model is not supported: {affine_model}. Please choose either 'wavenet' or 'simple_conv' instead."`.

Besides, This is a `ValueError`. Could you please replace `Exception` with `ValueError`? Thanks.

File: nemo/collections/tts/modules/common.py [hunk 2]
```diff
@@ -664,6 +667,8 @@ def get_scaling_and_logs(self, scale_unconstrained):
                 log_s_list.append(log_s_i[:, None])
             s = torch.cat(s_list, dim=1)
             log_s = torch.cat(log_s_list, dim=1)
+        else:
+            raise Exception("{} scaling fn not supported".format(scaling_fn))
```
Comment (XuesongYang) [inline nemo/collections/tts/modules/common.py:671]: same comment as above.

**Review by XuesongYang — CHANGES_REQUESTED** [review]
Thank you for your fix! Appreciate that.

File: nemo/collections/tts/modules/common.py (hunk 2 above)
Comment (XuesongYang) [reply nemo/collections/tts/modules/common.py:671]: `scaling_fn` --> `self.scaling_fn`

**Comment by XuesongYang** [comment]
close this PR. Please review https://github.com/NVIDIA/NeMo/pull/6177

**Pull request closed without merging**